Framework/Algorithms/test/PolarizationEfficiencyCorTest.h

```diff
@@ -515,7 +463,7 @@ class PolarizationEfficiencyCorTest : public CxxTest::TestSuite {
       alg->setProperty("P2", ws2);
       alg->setProperty("F1", ws3);
       alg->setProperty("F2", ws4);
-      alg->setPropertyValue("OutputWorkspace", "dummy");
+      alg->setPropertyValue("OutputWorkspace", OUTPUT_GRP_NAME);
```

Comment:
This comment should really be on line 458 but I can't add it there, sorry. It looks like there's some code duplication for setting up the `JoinISISPolarizationEfficiencies` algorithm. Could we move this into a helper method (i.e. similar to `createAlgorithm` except this one could be called `createJoinEfficienciesAlgorithm`)?